gossip-base/src/main/java/org/apache/gossip/manager/handlers/MessageHandlerFactory.java

```diff
@@ -35,8 +30,8 @@ public static MessageHandler defaultHandler() {
     return concurrentHandler(
         new TypedMessageHandler(Response.class, new ResponseHandler()),
         new TypedMessageHandler(ShutdownMessage.class, new ShutdownMessageHandler()),
-        new TypedMessageHandler(PerNodeDataMessage.class, new PerNodeDataMessageHandler()),
```

Comment:
Lets not remove the old system. Eventually I think the bulk only way makes sense but there is no issue in supporting both in the code and choosing with a switch